bigtop-tests/spec-tests/runtime/src/test/resources/testRuntimeSpecConf.groovy

```diff
@@ -300,7 +300,7 @@ specs {
       name = 'HADOOP_GETCONF'
       type = 'shell'
       arguments {
-        command = '[ `hdfs getconf -confKey dfs.permissions.superusergroup` == "hadoop" ]'
+        command = '[ `hdfs getconf -confKey dfs.permissions.superusergroup >/dev/null 2>/dev/null; echo $?` == "0" ]'
```

Comment:
The assumption here is that `hdfs getconf` will return non-zero code if the config key isn't present, right?